src/pybind/mgr/cephadm/module.py

```diff
@@ -1686,7 +1686,9 @@ def matches_network(host):
         r = False
 
         if daemon_type == 'osd':
-            return False if create_func(spec) else True # type: ignore
+            # TODO: return True would result in a busy loop
+            create_func(spec)
+            return False
```

Comment:
@bk201 what do you think?

Reply:
`return False` is confusing ... can we add a log line or comment as to why ...?

Reply:
moved to #35087